builder/src/Command/BuildCommand.php

```diff
@@ -0,0 +1,344 @@
+            foreach ($di2 as $appDir) {
+                if ($appDir->isDir()) {
+                    $apps[] = $appDir->getFilename();
+                }
+            }
+
+            $this->filesystem->mirror(
+                $docDir . '/' . $tag . '/src/',
+                $docDir . '/' . $tag . '/_merged/'
+            );
+
+            foreach ($apps as $app) {
+                $this->filesystem->mirror(
+                    $docDir . '/' . $tag . '/_generated/' . $app . '/doc/',
+                    $docDir . '/' . $tag . '/_merged/doc/_' . $app
+                );
+                $output->writeln(sprintf(
+                    'Merged app "%s" to %s',
+                    $app,
+                    realpath($docDir . '/' . $tag . '/_merged/doc/_' . $app)
+                ));
+            }
+
+            $this->compileFiles($tag, $projectDir, $docDir . '/' . $tag . '/_merged/doc', $output);
+
+            $this->runCommand(
+                ['pnpm', 'run', 'docusaurus', 'docs:version', $semver ? ($semver->major . '.' . $semver->minor) : $tag],
+                $projectDir,
+                $output
+            );
+        }
+
+        $orgConfig = file_get_contents($projectDir . '/docusaurus.config.ts');
+        $patchedConfig = str_replace('includeCurrentVersion: true', 'includeCurrentVersion: false', $orgConfig);
+        file_put_contents($projectDir . '/docusaurus.config.ts', $patchedConfig);
+
+        try {
+            $this->filesystem->mkdir($projectDir . '/build');
+            $process = $this->runCommand(
+                ['pnpm', 'run', 'build'],
+                $projectDir,
+                $output,
+                3600
+            );
+            file_put_contents(
+                $projectDir . '/build/build.html',
+                '<html lang="en"><pre>'.$process->getOutput().'</pre></html>'
+            );
+            file_put_contents(
+                $projectDir . '/build/build-error.html',
+                '<html lang="en"><pre>'.$process->getErrorOutput().'</pre></html>'
+            );
+            file_put_contents(
+                $projectDir . '/build/version.html',
+                sprintf(
+                    '<html lang="en"><pre>REFNAME:%s\nREFTYPE:%s\nDATETIME:%s</pre></html>',
+                    getenv('PHRASEA_REFNAME'),
+                    getenv('PHRASEA_REFTYPE'),
+                    getenv('PHRASEA_DATETIME')
+                )
+            );
+
+            file_put_contents($projectDir . '/docusaurus.config.ts', $orgConfig);
+
+            return Command::SUCCESS;
+        } catch (\Throwable $e) {
+            // Restore config to prevent side effects on next runs
+            $orgConfig = file_get_contents($projectDir . '/docusaurus.config.ts');
+            $patchedConfig = str_replace('includeCurrentVersion: false', 'includeCurrentVersion: true', $orgConfig);
+            file_put_contents($projectDir . '/docusaurus.config.ts', $patchedConfig);
```

Comment:
no need to re-open/re-patch the file (remove the 2 lines). `$orgConfig` still contains the original version
```suggestion
            file_put_contents($projectDir . '/docusaurus.config.ts', $orgConfig);
```